quickstart-postgresql/values.yaml

```diff
@@ -7,6 +7,12 @@ fullnameOverride: ''
 ## default set to 2 for smooth migration between nodes
 instances: 2
 
+## @param postgresMajor Major version of PostgreSQL to be used
+## This specifies the major version of PostgreSQL to deploy. 
+## Ensure compatibility with any extensions or configuration settings.
+## Allowed values are 15 and 17.
+postgresMajor: 17
```

Comment:
Do we need to first release APL with upgraded cnpg ?

Reply:
Yes. We decided to keep the imageCatalog out of the [APL-282 PR](https://github.com/linode/apl-core/pull/1784/files) but for this to work we need the clusterImageCatalog resource. Will add that back to the PR.